cmake/ChooseCudaCC.cmake

```diff
@@ -0,0 +1,89 @@
+#
+# after returning from this function, do not forget to call the following:
+#    set(RESULT_NAME ${RESULT_NAME} CACHE STRING "CUDA CC versions to compile")
+# replacing your own variable for RESULT_NAME
+#
+# We assume that MINCC default to 20
+# We assume that MINCUDAVERSION defaults to 7.0
+#
+function(ChooseCudaCC RESULT_NAME MINCC MINCUDAVERSION)
```

Comment:
even if it is case insensitive it is better to respect the convention and use `chooseCudaCC`

Reply:
done